openvdb_points/tools/AttributeArray.h

```diff
@@ -189,8 +189,9 @@ class AttributeArray
     typedef boost::shared_ptr<AccessorBase>             AccessorBasePtr;
 
 public:
-    enum Flag { TRANSIENT = 0x1, HIDDEN = 0x2, WRITEUNIFORM=0x8,
-                WRITEMEMCOMPRESS=0x10, WRITEDISKCOMPRESS=0x20, OUTOFCORE=0x40 };
+    enum Flag { TRANSIENT = 0x1, HIDDEN = 0x2, WRITESTRIDED=0x4, WRITEUNIFORM=0x8,
+                WRITEMEMCOMPRESS=0x10, WRITEDISKCOMPRESS=0x20, OUTOFCORE=0x40,
+                INTERLEAVED=0x80 };
```

Comment:
Could we maybe add some nice dox comments here to describe each of these flags and what they mean